src/test_files/test.py

```diff
@@ -27,15 +27,12 @@
 response = requests.get(my_url,
                         headers=headers,
                         params=params,
-                        verify=False)   # because of _ssl.c:1124 [SSL: CERTIFICATE_VERIFY_FAILED]
+                        verify=False)  # because of _ssl.c:1124 [SSL: CERTIFICATE_VERIFY_FAILED]
 
 packages_json = response.json()
-packages_str = json.dumps(packages_json)
-
-# print(response.status_code)
-# print(packages_json)
-# print(test_time)
 
 
-with open('data.json', 'w', encoding='utf-8') as f:  # data to json file
-    json.dump(packages_json, f, ensure_ascii=False, indent=4)   # file with nice markdown
+def to_file(filename):
```

Comment:
Do we still need this function?

Reply:
I also want to test client_waldur.py in this file, so I want to keep this function as by now.